examples/pidof.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
```

Comment:
Hi ..Thanks for quick response ..  I am newbie .. Please advice if I should use 
    # Copyright (c) 2009, Giampaolo Rodola' 
or 
   # Copyright (c) 2009, Karthik (my name) ?